core/tee/tee_obj.c

```diff
@@ -70,10 +70,11 @@ void tee_obj_close(struct user_ta_ctx *utc, struct tee_obj *o)
 
 void tee_obj_close_all(struct user_ta_ctx *utc)
 {
-	struct tee_obj_head *objects = &utc->objects;
+	struct tee_obj *o;
+	struct tee_obj *next;
 
-	while (!TAILQ_EMPTY(objects))
-		tee_obj_close(utc, TAILQ_FIRST(objects));
```

Comment:
Hi, Jerome,

Will the origin code use the pointer after free? I think about this, and can't find that. Can you explain something to me?